test/PositionManager.ts

```diff
@@ -131,10 +191,201 @@ describe("GammaPoolFactory", function () {
+                cfmm: cfmm.address,
+                amountsDesired: [10000, 100],
+                amountsMin: [1000, 10],
+                to: addr4.address,
+                protocol: 1,
+                deadline: ethers.constants.MaxUint256
+            }
+            const res = await (await posMgr.depositReserves(DepositReservesParams)).wait();
+            res.events.forEach(function(event: any, i: any){
+                expect(event.args).to.not.be.an('undefined');
+                console.log("event >>");
+                console.log(event.args);
+                expect(event.args.pool).to.equal(gammaPool.address);
+                expect(event.args.reservesLen).to.equal(4);
+                expect(event.args.shares.toNumber()).to.equal(18);
+            });
+        });
+
+        it("withdrawReserves", async function () {            
+            await gammaPool.approve(posMgr.address, ethers.constants.MaxUint256);//must approve before sending tokens
+
+            const WithdrawReservesParams =  {
+                cfmm: cfmm.address,
+                protocol: 1,
+                amount: 1000,
+                amountsMin: [100, 200, 300],
+                to: owner.address,
+                deadline: ethers.constants.MaxUint256
+            }
+
+            const res = await (await posMgr.withdrawReserves(WithdrawReservesParams)).wait();
+
+            res.events.forEach(function(event: any, i: any){
+                if(i == 0)
+                    return;
+                console.log("event >>");
+                console.log(event.args);
+                expect(event.args.pool).to.equal(gammaPool.address);
+                expect(event.args.reservesLen).to.equal(3);
+                expect(event.args.assets.toNumber()).to.equal(17);
+            });
+        });
     });
+
     // You can nest describe calls to create subsections.
     describe("Long Gamma", function () {
+        it("createLoan", async function () {
+            await cfmm.approve(posMgr.address, ethers.constants.MaxUint256);//must approve before sending tokens
+            
+            console.log("event >>");
+            console.log(createLoanArgs);
+            expect(createLoanArgs.pool).to.equal(gammaPool.address);
+            expect(createLoanArgs.tokenId.toNumber()).to.equal(19);
+        });
 
+        it("borrowLiquidity", async function () {
+            await cfmm.approve(posMgr.address, ethers.constants.MaxUint256);//must approve before sending tokens
+
+            const BorrowLiquidityParams = {
+                cfmm: cfmm.address,
+                protocol: 1,
+                tokenId: tokenId,
+                lpTokens: 1,
+                to: owner.address,
+                deadline: ethers.constants.MaxUint256
+            }
+            
+            const res = await (await posMgr.borrowLiquidity(BorrowLiquidityParams)).wait();
+            
+            const { args } = res.events[0]
+            console.log("event >>");
```

Comment:
Should probably use this same format to check the other events for sake of consistency. This format looks cleaner too

Reply:
Alright